include/dtpcontrols/WibulatorNode.hpp

```diff
@@ -12,28 +12,35 @@
 #define DTPCONTROLS_INCLUDE_DTPCONTROLS_WIBULATORNODE_HPP_
 
 // dtpcontrols Headers
+#include "dtpcontrols/toolbox.hpp"
 
 // uHal Headers
+
 #include "uhal/DerivedNode.hpp"
 #include "ers/Issue.hpp"
 
 // C++ Headers
-
+#include <vector>
+#include <fstream>
 
 namespace dunedaq {
   namespace dtpcontrols {
-    /**
-     * @brief      Wibulator firmware node
-     */
-    class WibulatorNode : public uhal::Node
-    {
 
-      UHAL_DERIVEDNODE(WibulatorNode)
+    class WibulatorNode : public uhal::Node {
 
+      UHAL_DERIVEDNODE(WibulatorNode)
+      
       public:
-      explicit WibulatorNode(const uhal::Node& node);
+      explicit WibulatorNode(const uhal::Node& );
```

Comment:
please ensure all method arguments are named in the header.  don't leave it to the user to guess what they mean.